PointsOfInterest/aed.csv

```diff
@@ -0,0 +1,34 @@
+name,description,note,latitude,longitude
+,Between the male and female washrooms (across from the front desk),Columbia Lake Village (CLV),43.47082,-80.56244

```

Comment:
Good practice however as a colleague mentioned to me is to not use acronyms when introducing something the first time.